parameters/world_modeling/people_recognition_3d.yaml

```diff
@@ -0,0 +1,15 @@
+recognize_people_srv_name: "detect_people"
+probability_threshold: 0.2
+link_threshold: 0.5
+heuristic: "shoulder"
+arm_norm_threshold: 0.5
+neck_norm_threshold: 0.7
+wave_threshold: 0.2
+vert_threshold: 0.7
+hor_threshold: 0.4
+padding: 5
+enable_topic_mode: false
+
+camera_info_depth: "hero/head_rgbd_sensor/depth_registered/camera_info"
```

Comment:
Absolute topics should start with a slash

Reply:
True that